documentation/design-docs/dotnet-monitor.yaml

```diff
@@ -0,0 +1,457 @@
+        default: "30"
+      - name: "configuration"
+        in: body
+        schema:
+          type: object
+          items:
+            $ref: "#/definitions/EventPipeConfiguration"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /trace/{uid}:
+    post:
+      tags:
+      - "traces"
+      operationId: "posttrace"
+      produces:
+      - "application/octet-stream"
+      parameters:
+      - name: "duration"
+        in: "query"
+        description: "Duration of the trace"
+        required: false
+        type: "integer"
+        default: "30"
+      - name: "uid"
+        in: "path"
+        description: "ID of the process to collect a process dump"
+        required: true
+        type: "string"
+      - name: "configuration"
+        in: body
+        schema:
+          type: object
+          items:
+            $ref: "#/definitions/EventPipeConfiguration"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /logs/:
+    get:
+      tags:
+      - "logs"
+      operationId: "getdefaultlogs"
+      produces:
+      # Newline-delimited JSON- http://www.ietf.org/rfc/rfc4288.txt
+      - "application/x-ndjson"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+  /logs/{uid}:
+    get:
+      tags:
+      - "logs"
+      operationId: "getlogs"
+      produces:
+      - "application/x-ndjson"
+      parameters:
+      - name: "uid"
+        in: "path"
+        description: "Process ID of the process to collect a process dump"
+        required: true
+        type: "string"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+    post:
+      tags:
+      - "logs"
+      operationId: "postlogs"
+      produces:
+      - "application/x-ndjson"
+      parameters:
+      - name: "uid"
+        in: "path"
+        description: "Process ID of the process to collect a process dump"
+        required: true
+        type: "string"
+      - name: "configuration"
+        in: "body"
+        schema:
+          type: "array"
+          items:
+            $ref: "#/definitions/LogFilter"
+      responses:
+        200:
+          description: "Success"
+          schema:
+            format: "byte"
+        400:
+          description: "Bad request"
+          schema:
+            $ref: "#/definitions/Error"
+            
+  /metrics/:
+    get:
```

Comment:
We should consider producing metrics on json format as well.

Reply:
Added `application/json` to the endpoints